src/lib/units/priorities.js

```diff
@@ -1,13 +1,18 @@
 var priorities = {};
 
-export function addUnitPriority(unit, priority) {
-    priorities[unit] = priority;
+export function addUnitPriority(unit, priority, getterSetter) {
+    priorities[unit] = {priority: priority, getSet: getterSetter};
 }
 
 export function getPrioritizedUnits(unitsObj) {
     var units = [];
     for (var u in unitsObj) {
-        units.push({unit: u, priority: priorities[u]});
+        units.push({
+            unit: u,
+            value: unitsObj[u],
+            priority: priorities[u].priority,
+            getSet: priorities[u].getSet
+        });
```

Comment:
All the getter/setter functions get registered here, in addition to registering them directly on the Moment prototype as before, so that we can access them in the generic `.get()` function (aka "stringSet", where I left the other comment above).